scripts/nbsite_cleandisthtml.py

```diff
@@ -31,20 +28,29 @@ def IGetFiles(d):
 for folder in (".doctrees", "_sources"):
     d = os.path.join(htmldir,folder)
     try:
-        print("removing %s"%d)
-        shutil.rmtree(d)
+        if dry_run:
+            print("would remove %s"%d.split(output)[-1])
+        else:
+            print("removing %s"%d)
+            shutil.rmtree(d)
```

Comment:
I'm a bit confused; shouldn't each of these be something like:

```
file = "%s" % d.split(output)[-1]
if dry_run:
   print("would remove %s" % file)
else:
    print("removing %s" % file)
    shutil.rmtree(d)
```

I.e. why is the file bit different for dry_run?

Reply:
Basically I couldn't decide. I thought it was easier to read with just the filename. If you agree I can make them both that way.

Reply:
I haven't seen the output, but whatever you choose, I'd request making it the same in the two conditions.  Also please put spaces around % used as a modulo operator, so that it doesn't look like a %d format specifier.

Reply:
I cleaned it up so that there is the full path at the top an then the rel paths from there:

```
Removing files from: /home/travis/build/pyviz/nbsite/tmpdoctest/_build/html
removing .doctrees
removing _sources
removing objects.inv
removing /home/travis/build/pyviz/nbsite/tmpdoctest/_build/html/getting_started/1-Introduction_temp_evaluated.ipynb
removing /home/travis/build/pyviz/nbsite/tmpdoctest/_build/html/getting_started/2-Installation_temp_evaluated.ipynb
removing /home/travis/build/pyviz/nbsite/tmpdoctest/_build/html/getting_started/3-Tabular_Datasets_temp_evaluated.ipynb
removing /home/travis/build/pyviz/nbsite/tmpdoctest/_build/html/getting_started/4-Gridded_Datasets_temp_evaluated.ipynb
removing /home/travis/build/pyviz/nbsite/tmpdoctest/_build/html/user_guide/01-Annotating_Data_temp_evaluated.ipynb
removing /home/travis/build/pyviz/nbsite/tmpdoctest/_build/html/user_guide/03-Customizing_Plots_temp_evaluated.ipynb
removing /home/travis/build/pyviz/nbsite/tmpdoctest/_build/html/user_guide/11-Responding_to_Events_temp_evaluated.ipynb
```

Reply:
Sounds good, though it looks to me like the full paths are still being shown in the last 7 lines?

Reply:
Also, I guess I'm still confused about how this will work in practice, a confusion that predates you working on this.  At least on my own system, don't I *want* the evaluated ones to hang around, so that the next time I build the docs, the notebooks don't have to be re-run?  I would have thought that they just need to stop being named crazy filenames like _temp_evaluated.ipynb; why not just leave them there, and have the normal .ipynb filename?

Reply:
There is probably some step in this workflow that I don't know about that makes this make sense, so sorry if this is just noise.

Reply:
Ah, the output has now been updated above; looks good.

Reply:
> Also, I guess I'm still confused about how this will work in practice, a confusion that predates you working on this. At least on my own system, don't I want the evaluated ones to hang around, so that the next time I build the docs, the notebooks don't have to be re-run? 

@jbednar I can describe what used to happen; I don't know what happens now (if there've been recent relevant changes). Repeated build steps would only re-run a notebook if that notebook was changed. Sphinx itself was providing the make-like "re-run only if the input file has been touched" bit. That all sounds good. The problem was that the run/evaluated notebook was always saved as notebook_temp_evaluated.ipynb (or something like that) in the html output directory. 

> I would have thought that they just need to stop being named crazy filenames like _temp_evaluated.ipynb; why not just leave them there, and have the normal .ipynb filename?

I was planning to be able to specify location of 'source notebooks', location of 'build notebooks', and location of (html) 'output'. I guess I was thinking that some too-expensive-to-run-on-CI notebooks might be stored already run in a git repository different from the source git repository. But I hadn't really thought it through properly or decided how to organize things overall.